server/core/region.go

```diff
@@ -605,12 +607,27 @@ func (r *RegionsInfo) GetOverlaps(region *RegionInfo) []*RegionInfo {
 
 // AddRegion adds RegionInfo to regionTree and regionMap, also update leaders and followers by region peers
 func (r *RegionsInfo) AddRegion(region *RegionInfo) []*RegionInfo {
-	// Add to tree and regions.
-	overlaps := r.tree.update(region)
-	for _, item := range overlaps {
-		r.RemoveRegion(r.GetRegion(item.GetID()))
+	var overlaps []*RegionInfo
+	treeNeedAdd := true
```

Comment:
Could you add a comment about the variable.

Reply:
Done, thanks.